CHANGELOG.md

```diff
@@ -1,6 +1,20 @@
 CHANGELOG
 =========
 
+### 4.0.0 (2016-xx-xx)
+
+* Added: Interface for `Geocoder\Model\AddressCollection` called `Geocoder\GeocoderResult`. Public APIs are updated to type hint for `Geocoder\GeocoderResult`.
+* Added: Interface for `Geocoder\Model\Address` called `Geocoder\Location`. Public APIs are updated to type hint for `Geocoder\Location`.
+* Added: `Country::isDefined`
+* Added: `Cordinates::isDefined`
+* Changed: `Location::getCoordinates`, `Location::getBounds` and `Location::getCountry` will never return null.
+* Removed: `AdminLevel::toString` in favor for `AdminLevel::__toString`.
+* Removed: `Country::toString` in favor for `Country::__toString`.
+* Removed: `Address::getCountryCode` in favor for `Address::getCountry()->getCode()`.
+* Removed: `Address::getLongitude` in favor for `Address::getCoordinates()->getLongitude()`.
+* Removed: `Address::getLatitude` in favor for `Address::getCoordinates()->getLatitude()`.
```

Comment:
I agree that a *shorter way* would be useful. But do we really need both of these: 
* `Address::getCoordinates()->getLatitude()`
* `Address::getLatitude()`

I would like to drop one of them. I chose the latter because I liked the separation of coordinates into a separate object. But Im happy to change if you prefer.